tests/jballerina-unit-test/src/test/resources/test-src/object/object_langlib_function_invocation_negative.bal

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.'object as lang_obj;
+import ballerina/lang.runtime;
+
+class Person {
+    string name = "";
+    int age = 10;
+}
+
+public function testLangLibFuncInvocationNegative() {
+    Person person = new;
+    string str = person.toString();
+}
+
+class Employee {
+    string name = "";
+    int age = 10;
+
+    public function toString() returns string {
+        return "Name:" + self.name + " age:" + self.age.toString();
+    }
+}
+
+public function testObjectMethodInvocation() {
+    Employee emp = new;
+    string str = emp.toString();
+}
+
+public type Frame readonly & object {
+   public function returnString() returns string;
+};
+
+public readonly class FrameImpl {
+   *Frame;
+}
+
+public class RawTemplateImpl {
+    *lang_obj:RawTemplate;
+
+    public function init() {
+        self.insertions = [1, 2, 3];
+    }
+}
+
+public class DynamicListenerImpl {
+    *runtime:DynamicListener;
+
+    public function gracefulStop() returns error? {
+        return ();
+    }
+
+    public function immediateStop() returns error? {
+        return ();
+    }
+}
+
+public class DynamicListenerImpl2 {
+    *runtime:DynamicListener;
+
+    public function 'start() returns error? {
+        return ();
+    }
+
+    public function gracefulStop() returns error? {
+        return ();
+    }
+
+    public function immediateStop() returns error? {
+        return ();
+    }
+}
+
+public readonly class StackFrameImpl {
+    *runtime:StackFrame;
+}
+
+public readonly class StackFrameImpl2 {
```

Comment:
Here onward, it's a positive test right? Shall we move this to a separate file?

And IMO it will be better if we can introduce a separate project and use constructs from that rather than (or in addition to) using langlib modules like we do with other bala tests, so that the user-defined project path is also tested.

We can do this in a separate PR.

Reply:
Refactored to separate files and improved and enabled bala test in 2a8045f1513eb0a447d358b96b77ad6ab656a715